src/orquestra/sdk/_base/cli/_dorq/_login/_login.py

```diff
@@ -61,7 +61,7 @@ def _on_cmd_call_with_exceptions(
     def _prompt_for_login(self, url: str, ce: bool):
         try:
             asyncio.run(self._get_token_from_server(url, ce, 60))
-        except web.GracefulExit:
+        except (web.GracefulExit, KeyboardInterrupt):
```

Comment:
**question**: should we tell the user what's the command to run to set the token manually?

Reply:
This is handled in `self._login_presenter.prompt_for_login(self._login_url, url, ce)`.

<img width="725" alt="Screenshot 2023-02-10 at 17 45 09" src="https://user-images.githubusercontent.com/70290797/218160268-78b02195-fc55-4ded-afa9-6f7d93c96cc9.png">